changelog.txt

```diff
@@ -25,6 +26,10 @@ that repo.
 -@ `unforbid`: fix detection of unreachable items for items in containers
 
 ## Misc Improvements
+- `troubleshoot-item`: output as bullet point list with indenting, with item description and ID at top. Now reports on more data, regarding items that are:
+    - hidden; artifacts; held by a unit, with unit name printed
+    - in a storage item or container, displaying the type of container
+    - are containers, with the contents of the container summarised with count of each item type
```

Comment:
Unfortunately nested lists don't work like this in the changelog. Any item, whether immediately after the tool name or on a separate line, is treated as an individual change for that tool, and then grouped later when the HTML is generated. Order is typically maintained, but it looks like this:
![image](https://user-images.githubusercontent.com/3719547/221061249-07a4f498-6cf9-4095-b507-f5489a5c943a.png)

I think the "Now reports on more data" bit could be its own item, including the three list items currently under it. Does that change work for you?

Reply:
fixed in https://github.com/DFHack/scripts/commit/e70393ff2132e2f5d00ffeb154e8d9242f89a284

Reply:
Thanks, both! A single ine would have been fine but if you're happy with Myk's edit then I am too.